src/Features/Core/Portable/ExtractMethod/MethodExtractor.Analyzer.cs

```diff
@@ -117,10 +123,17 @@ public async Task<AnalyzerResult> AnalyzeAsync()
                     model, symbolMap, parameters, failedVariables, unsafeAddressTakenUsed, returnTypeHasAnonymousType);
 
                 return new AnalyzerResult(
-                    newDocument,
-                    typeParametersInDeclaration, typeParametersInConstraintList,
-                    parameters, variableToUseAsReturnValue, returnType, awaitTaskReturn,
-                    instanceMemberIsUsed, endOfSelectionReachable, operationStatus);
+                    document: newDocument,
+                    typeParametersInDeclaration: typeParametersInDeclaration,
+                    typeParametersInConstraintList: typeParametersInConstraintList,
+                    variables: parameters,
+                    variableToUseAsReturnValue: variableToUseAsReturnValue,
+                    returnType: returnType,
+                    awaitTaskReturn: awaitTaskReturn,
+                    instanceMemberIsUsed: instanceMemberIsUsed,
+                    shouldBeReadOnly: shouldBeReadOnly,
+                    endOfSelectionReachable: endOfSelectionReachable,
```

Comment:
most of these named parameters seem redundant.

Reply:
Sure, happy to remove if you feel they add clutter.